source/languages/en/riak/theory/concepts/crdts.md

```diff
@@ -235,7 +235,7 @@ Data Type | Convergence rule
 :--------|:------------
 Flags | `enable` wins over `disable`
 Registers | The most chronologically recent value wins, based on timestamps
-Counters | Each actor keeps an independent count for increments and decrements; upon merge, the pairwise maximum of the counts for each actor will win (e.g. if one count for an actor holds 172 and the other holds 173, 173 will win upon merge)
+Counters | Implemented as a PN-Counter ([paper](http://hal.upmc.fr/docs/00/55/55/88/PDF/techreport.pdf)), so all increments and decrements by all actors are eventually applied.  Everyone wins.
```

Comment:
done in https://github.com/locationlabs/basho_docs/commit/9018c201ca04b9e7823e2c7894a67bc07aa46543